server/community/queries.ts

```diff
@@ -116,3 +124,78 @@ export const updateCommunity = (inputValues, updatePermissions) => {
 		return filteredValues;
 	});
 };
+
+export const getCountOfPubsOrCollections = async (userId: string, communityId: string) => {
```

Comment:
A name that better describes what this does might be `isUserAffiliatedWithCommunity`